### PR TITLE
retrieve only latest session in SQLTrackerStore

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- retrieve only latest session in `SQLTrackerStore`

**Status (please check what you already did)**:
- [x] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: rasa/core/tracker_store.py [hunk 1]
```diff
@@ -706,15 +706,39 @@ def keys(self) -> Iterable[Text]:
...
         """Create a tracker from all previously stored events."""
 
+        import sqlalchemy as sa
+        from rasa.core.events import SessionStarted
+
         with self.session_scope() as session:
-            query = session.query(self.SQLEvent)
-            result = (
-                query.filter_by(sender_id=sender_id)
+            # Subquery to find the timestamp of the first `SessionStartedEvent`.
```
Comment (ricwo) [inline rasa/core/tracker_store.py]:
```suggestion
            # Subquery to find the timestamp of the first `SessionStarted` event
```

File: setup.py [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "aioresponses~=0.6.0",
     "moto~=1.3.8",
     "fakeredis~=1.0",
+    "mongomock~=3.18.0",
```
Comment (wochinge) [inline setup.py]: not `~=3.18`?

Comment (ricwo) [reply setup.py]: I believe they're the same thing 
https://jubianchi.github.io/semver-check/#/~%3D3.18.0/3.18.23

Comment (ricwo) [reply setup.py]: I believe they're the same thing
https://jubianchi.github.io/semver-check/#/~%3D3.18.0/3.18.23

Comment (wochinge) [reply setup.py]: Nope, https://www.python.org/dev/peps/pep-0440/#compatible-release

Comment (ricwo) [reply setup.py]: 👍

Comment (ricwo) [inline setup.py]:
```suggestion
    "mongomock~=3.18",
```

**Pull request merged**